doc/cookbook/curl-mock/CurlMock.lhs

```diff
@@ -130,24 +130,12 @@ generateCurl :: (GenerateList Mocked (Foreign Mocked api), HasForeign NoLang Moc
 generateCurl p host =
   fmap T.unlines body
   where
-  body = foldr (\endp curlCalls -> mCons (generateEndpoint host endp) curlCalls) (return [])
+  body = mapM (generateEndpoint host)
```

Comment:
I guess most people dealing with servant will know mapM. The mCons helper function adds unrelated complexity to the recipe.